ingester/src/server/grpc/query.rs

```diff
@@ -116,10 +120,20 @@ impl<Q> FlightService<Q> {
             )
             .recorder(&[]);
 
+        let query_request_frame_encoding_duration = Arc::new(
+            metrics
+                .register_metric::<DurationHistogram>(
+                    "query_request_frame_encoding_duration",
```

Comment:
Lets make sure we don't confuse anyone!

```suggestion
                    "ingester_query_request_frame_encoding_duration",
```

Reply:
done